packages/jest-reporters/src/GitHubActionsReporter.ts

```diff
@@ -42,7 +45,7 @@ type PerformanceInfo = {
 
 type ResultTreeLeaf = {
   name: string;
-  passed: boolean;
+  passed: Status;
```

Comment:
Perhaps this prop should be renamed to `status` or so?

This is only used internally, right? Just a detail, but makes it more understandable for readers.